tests/callbacks/test_early_stopping.py

```diff
@@ -344,3 +344,56 @@ def validation_epoch_end(self, outputs):
 def test_early_stopping_mode_options():
     with pytest.raises(MisconfigurationException, match="`mode` can be .* got unknown_option"):
         EarlyStopping(mode="unknown_option")
+
+
+@pytest.mark.parametrize(
```

Comment:
@SeanNaren What are these parameters for?

Reply:
woops, let me remove